code/_onclick/hud/_defines.dm

```diff
@@ -128,11 +128,13 @@
 #define ui_gloves "WEST+2:10,SOUTH+1:7"
 
 
-#define ui_glasses "WEST:6,SOUTH+2:9"
+#define ui_neck "WEST:6,SOUTH+3:11"
 #define ui_mask "WEST+1:8,SOUTH+2:9"
 #define ui_l_ear "WEST+2:10,SOUTH+2:9"
 #define ui_r_ear "WEST+2:10,SOUTH+3:11"
 
+#define ui_glasses "WEST:6,SOUTH+2:9"
+
 #define ui_head "WEST+1:8,SOUTH+3:11"
```

Comment:
I feel like these should be the other way around if anything, but this still works fine as-is.
```suggestion
#define ui_glasses "WEST:6,SOUTH+2:9"
#define ui_mask "WEST+1:8,SOUTH+2:9"
#define ui_l_ear "WEST+2:10,SOUTH+2:9"
#define ui_r_ear "WEST+2:10,SOUTH+3:11"

#define ui_neck "WEST:6,SOUTH+3:11"

#define ui_head "WEST+1:8,SOUTH+3:11"
```